feature-libs/quote/styles/_quote-links.scss

```diff
@@ -24,6 +24,18 @@
             border: none;
             padding-inline-end: 0;
           }
+
+          button {
+            color: var(--cx-color-background-dark);
+
+            &:hover {
+              color: var(--cx-color-background-dark);
+            }
+
+            &:focus {
+              border-color: var(--cx-color-primary);
+            }
```

Comment:
@suprishi, styling for focus should be added, otherwise no border is displayed when one tabs within the quote oveview